string prop_type
float32 theta_1
float32 theta_2
float32 closest_pnt_dist
float32 closest_pnt_angle

